composeApp/src/commonMain/composeResources/values/strings.xml

```diff
@@ -1,2 +1,22 @@
 <resources>
+    <string name="app_name">Minesweeper</string>
+
+    <string name="apply">Apply</string>
+    <string name="cancel">Cancel</string>
+    <string name="mine_count">Mine count</string>
+    <string name="width">Width</string>
+    <string name="height">Height</string>
+    <string name="restart">Restart</string>
+    <string name="settings">Settings</string>
+
+    // TODO: Check how Accessibility work, if its valueble - change trailing for: %d mines left.
+    // TODO: Accessibility check gives info that user can't select @Composable Counter`s with tab-navigation in our app - need to undertsand why and fix it.
```

Comment:
Could you please elaborate on this comment? What does it mean?

Reply:
Its more easy to show, check this out: https://youtu.be/_QhpFqwDvp0

You can also try VoiceOver on your macOS system: press cmd + F5 (mic button).
When i pressing tab in app, VoiceOver navigating us per elements, u can hear:
Settings button
Restart button
unknown <- its element of game field of mines (one more idea: user can access each cell with tab)

And we doesn't navigated (when pressing `tab` button) to Counter's and not hear information that it has.

Reply:
This makes sense, but should this comment be here? Let's just remove it for now. We could just follow-up with a fix at some point later.